contracts/examples/FlatDirectory.sol

```diff
@@ -46,4 +46,8 @@ contract FlatDirectory is W3RC3 {
         require(msg.sender == owner, "must from owner");
         selfdestruct(payable(owner));
     }
+
+    function balanceOf(address addr)public view returns(uint256){
```

Comment:
Looks like the method is not used in any contract other than tests.  We should remove it if it is not be used in prod contract.

Further, we could use other ways to find the balance in tests such as https://docs.ethers.io/v4/cookbook-accounts.html.